content/blog/2018/12/2018-12-14-java11-preview-availability.adoc

```diff
@@ -14,6 +14,14 @@ links:
   sig: platform
 ---
 
+[WARNING]
+--
+Guidelines in this blogpost are rendered obsolete link:/blog/2019/02/04/running-jenkins-with-java-11/[by the Java 11 Support Availability
```

Comment:
https://jenkins.io/doc/administration/requirements/java/ would need to be updated as well